editor/store/test/effects.js

```diff
@@ -346,6 +346,7 @@ describe( 'effects', () => {
 					id: 'SAVE_POST_NOTICE_ID',
 					isDismissible: true,
 					status: 'success',
+					spokenMessage: 'Post published!',
```

Comment:
These should be localized.

Reply:
Actually, these are only tests. The message is localized :)